core/src/types.ts

```diff
@@ -55,6 +55,8 @@ export declare interface CollateralRow extends CollateralConfig, Partial<MakerPa
 
 export declare interface AuctionTransaction extends Auction, TransactionFees {
     transactionProfitMinusFees: BigNumber;
+    totalFeeDAI: BigNumber;
+    totalFeeETH: BigNumber;
```

Comment:
Can you please show this number in the transaction table instead of the `biddingTransactionFee`?